### PR TITLE
feature: Applet reports updates (M2-7254)

### PR DESCRIPTION
**Pull request by andrevitalb**
- [x] Tests for the changes have been added
- [ ] Delivered the `fix` or `feature` branches into `develop` or `release` branches via `Squash and Merge` (to keep clean history)

### 📝 Description

🔗 [Jira Ticket M2-7254](https://mindlogger.atlassian.net/browse/M2-7254)

This PR adds the following fields to both `activity_user_journey.csv` & `report.csv`:

- `source_user_secret_id`
- `source_user_nickname`
- `source_user_relation`
- `target_user_secret_id`
- `target_user_nickname`
- `target_user_tag`
- `input_user_subject_id`
- `input_user_secret_id`
- `input_user_nickname`
- `source_user_tag`

It also does three field renames:

- `source_subject_id ` ->  `source_user_subject_id`
- `target_subject_id ` ->  `target_user_subject_id`

### CODE REVIEW

**Comment by ChaconC** [comment]
Please review failing tests.

File: src/shared/consts.tsx [hunk 1]
```diff
@@ -324,7 +324,19 @@ export const reportHeader = [
   'activity_end_time',
   'flag',
   'secret_user_id',
-  'userId',
+  'user_id',
```
Comment (ChaconC) [inline src/shared/consts.tsx]: This should be `userId` as it will be a breaking change for scripts running on data exports.

Comment (andrevitalb) [reply src/shared/consts.tsx]: Wouldn't the other renames break those scripts as well?
- `source_subject_id` -> `source_user_subject_id`
- `target_subject_id` -> `target_user_subject_id`

Comment (ChaconC) [reply src/shared/consts.tsx]: These fields are accounted for in the current communication about changes to the report file.

**Review by ChaconC — DISMISSED** [review]

**Review by ChaconC — APPROVED** [review]

**Pull request merged**